src/main/java/crimson/ResultEmailer.java

```diff
@@ -0,0 +1,48 @@
+package crimson;
+
+import com.mashape.unirest.http.HttpResponse;
+import com.mashape.unirest.http.JsonNode;
+import com.mashape.unirest.http.Unirest;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import karmosin.ContinuousIntegrationJob;
+
+public class ResultEmailer{
```

Comment:
this needs to specify it is implementing `karmosin.ResultEmailer`

Reply:
it is specified now